app/src/main/java/com/nutomic/syncthingandroid/views/FoldersAdapter.java

```diff
@@ -118,17 +123,15 @@ public View getView(int position, View convertView, @NonNull ViewGroup parent) {
      */
     private static String getLocalizedState(Context c, String state, int percentage) {
         switch (state) {
-            case "idle":     return c.getString(R.string.state_idle);
-            case "scanning": return c.getString(R.string.state_scanning);
-            case "cleaning": return c.getString(R.string.state_cleaning);
-            case "syncing":  return c.getString(R.string.state_syncing, percentage);
-            case "error":    return c.getString(R.string.state_error);
-            case "unknown":  // Fallthrough
-            case "":         return c.getString(R.string.state_unknown);
-        }
-        if (BuildConfig.DEBUG) {
-            throw new AssertionError("Unexpected folder state " + state);
+            case "idle":        return c.getString(R.string.state_idle);
+            case "scanning":    return c.getString(R.string.state_scanning);
+            case "cleaning":    return c.getString(R.string.state_cleaning);
+            case "syncing":     return c.getString(R.string.state_syncing, percentage);
+            case "error":       return c.getString(R.string.state_error);
+            case "unknown":     // Fallthrough
+            case "":            return c.getString(R.string.state_unknown);
```

Comment:
When is this an empty string?

Reply:
Perhaps just replace it with `default: return state`, and return unknown explicitly?

Reply:
According to my debugging, the string model.state is empty when a folder is in paused state. The REST API does not return "paused" here.

Reply:
Sure, but that's just a coincidence, I am pretty sure that's not intended, so deliberately checking for that seems wrong.